extensions/brew/src/components/list.tsx

```diff
@@ -59,14 +58,16 @@ export function FormulaListItem(props: {
   let version = formula.versions.stable;
   let tintColor = Color.SecondaryText;
```

Comment:
I wonder if we could use a lighter tint color here. For example this works quite nicely - it's closer to the tertiary text color use for the item's subtitle:

```
const tertiaryText: Color.Dynamic = {
  light: "#00000066",
  dark: "#FFFFFF66"
}
```